sdk/python/kfp/components/tasks_group.py

```diff
@@ -188,8 +188,13 @@ def __init__(
         self,
         items: Union[for_loop.ItemList, pipeline_channel.PipelineChannel],
         name: Optional[str] = None,
+        parallelism: Optional[int] = 0,
```

Comment:
WDYT about using the same `= None` followed by `parallelism = parallelism or 0` as discussed in v1 [here](https://github.com/kubeflow/pipelines/pull/8112#discussion_r940488476), that way there is no use code change needed for v1 -> v2 migration?

Reply:
Good idea. Fixed